numpy/f2py/crackfortran.py

```diff
@@ -930,7 +942,18 @@ def analyzeline(m, case, line):
             block = 'block data'
         if re.match(r'python\s*module', block, re.I):
             block = 'python module'
-        name, args, result, bind = _resolvenameargspattern(m.group('after'))
+        if block == 'type':
+            name, attributes, params = _resolvetypedefpattern(m.group('after'))
+            for att in attributes:
+                if att == 'public' or att == 'private':
+                    groupcache[groupcounter]['vars'][name] = {'attrspec': [att, ]}
+                else:
+                    outmess('analyzeline: ignored type definition attribute "%s"\n' % att.replace('@',''))
+            args = []
+            result = None
+            bind = None
```

Comment:
I think we can get rid of the above three lines.

Reply:
In fact, `args` and `result` are used later in the routine. You're right about `bind`, it is used only for entries, not for types. So I've removed it. Thanks.